cmd/fortifyExecuteScan.go

```diff
@@ -380,15 +380,17 @@ func generateAndDownloadQGateReport(config fortifyExecuteScanOptions, sys fortif
 	return data, nil
 }
 
-func checkArtifactStatus(config fortifyExecuteScanOptions, sys fortify.System, projectVersionID int64, buildLabel string, filterSet *models.FilterSet, artifact *models.Artifact, numInvokes int) error {
-	numInvokes++
+var errProcessing = errors.New("artifact still processing")
```

Comment:
It is intended to be a constant, but this does not compile. The return value of errors.New() needs to be a constant, which it is not. I don't really understand why I cannot assign a var to a const.

Reply:
BTW: https://medium.com/@smyrman/writing-constant-errors-with-go-1-13-10c4191617
Looks a bit like it's not really worth the trouble, to be honest. As I take it, this would have to be written out somewhere in each package. Maybe there is a more recent solution, as that is for Go 1.13.